files/ansible/tasks/install_perf.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install perf on Debian
+  apt:
+    name: linux-perf
+    state: present
+    update_cache: yes
+  when: ansible_facts.os_family == 'Debian'
+
+- name: Install perf on Ubuntu
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - linux-tools
+    - linux-tools-generic
+    - linux-tools-common
+  when: ansible_facts.os_family == 'Ubuntu'
+
+- name: Install perf on CentOS
+  yum:
+    name: perf
+    state: present
+    update_cache: yes
+  when: ansible_facts.os_family == 'RedHat'
```

Comment:
CentOS and Suse can be on the same task

Reply:
Same package name, different package manager.

Reply:
Right....